overlord/restart/restart.go

```diff
@@ -426,6 +426,16 @@ func PendingForChange(st *state.State, chg *state.Change) bool {
 	return rm.pendingForSystemRestart(chg)
 }
 
+// PendingTypeForChange returns the type of restart pending for the given change
+// or returns RestartUnset if none is pending.
+func PendingTypeForChange(chg *state.Change) RestartType {
```

Comment:
is this used somewhere?

Reply:
Nope, it was leftovers from testing, it's removed again, really nice catch